frontend-new/pages/main/index.tsx

```diff
@@ -1,78 +1,59 @@
-import { useCallback, useMemo, useState } from 'react'
-import useSWR from 'swr'
-import styled from '@emotion/styled'
-import { LinkCreationForm } from './components/LinkCreationForm'
-import { LinkList } from './components/LinkList'
-import { ResponseContainer, ResponseType } from './components/ResponseContaner'
-import { Search } from './components/Search'
-import { fetcher } from '../../utils/fetcher'
-import { Link, LinkCreate, LinkCreateResponse } from '../../types'
-import { find } from 'lodash'
-
-const StyledDiv = styled.div`
-  background-color: #fff;
-  height: calc(100% - 64px);
-  padding: 64px 200px 0;
-
-  .scrollable {
-    height: calc(100% - 176.5px);
-    overflow: scroll;
-  }
-`
-interface NotificationState {
-  link: Link
-  type: ResponseType
-  message: string
+import { Box } from '@mui/material'
```

Comment:
There shouldn't be any component files unded `pages`, this is a routing folder and is meant to add files for the pages that can be navigated to. 

Move all the components to the componenbts folder, and here just add the `index` file with the main dashboard page.

Reply:
okay I see, left just import/export here